src/containers/AskMeBot/AskMeBot.tsx

```
@@ -0,0 +1,213 @@
+              <span className={styles.LogoContainer}>
+                <Logo />
+              </span>
+              Ask Glific
+            </span>
+            <CloseIcon onClick={() => setOpen(false)} />
+          </div>
+          <div className={styles.Messages}>
+            {messages
+              .filter((i) => !i.prompt)
+              .map((message) => (
+                <div className={`${message.role === 'system' ? styles.System : styles.User}`}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)